Dang, looks like that broke a test
```
   ======================================================================
  ERROR: corehq.apps.app_manager.tests.test_management_commands:MigrateCaseSearchPromptItemsetIdsTest.test_migrate
  ----------------------------------------------------------------------
  Traceback (most recent call last):
    File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_management_commands.py", line 48, in test_migrate
      itemset = self._migrate_property(CaseSearchProperty(
    File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_management_commands.py", line 42, in _migrate_property
      return update.modules[0].search_config.properties[0].itemset
  AttributeError: 'dict' object has no attribute 'modules'
  
  ----------------------------------------------------------------------
  ```

--- a/corehq/apps/app_manager/management/commands/migrate_case_search_prompt_itemset_ids.py
+++ b/corehq/apps/app_manager/management/commands/migrate_case_search_prompt_itemset_ids.py
@@ -22,7 +22,8 @@ class Command(AppMigrationCommandBase):
                 if not isinstance(properties, list):
                     continue
                 for prop in properties:
-                    (new_itemset, should_save) = wrap_itemset(prop.get('itemset'))
+                    (new_itemset, updated) = wrap_itemset(prop.get('itemset'))
+                    should_save = should_save or updated
                     prop['itemset'] = new_itemset
 
         return app_doc if should_save else None

--- a/corehq/apps/app_manager/tests/test_management_commands.py
+++ b/corehq/apps/app_manager/tests/test_management_commands.py
@@ -10,6 +10,7 @@ from corehq.apps.app_manager.models import (
     Itemset,
     Module,
 )
+from corehq.apps.app_manager.util import get_correct_app_class
 
 
 class MigrateCaseSearchPromptItemsetIdsTest(SimpleTestCase):
@@ -37,9 +38,10 @@ class MigrateCaseSearchPromptItemsetIdsTest(SimpleTestCase):
 
     def _migrate_property(self, prop):
         self.doc['modules'][0]['search_config']['properties'] = [prop.to_json()]
-        update = Command().migrate_app(self.doc)
-        if update:
-            return update.modules[0].search_config.properties[0].itemset
+        app_doc = Command().migrate_app(self.doc)
+        if app_doc:
+            app = get_correct_app_class(app_doc).wrap(app_doc)
+            return app.modules[0].search_config.properties[0].itemset
 
     def test_text(self):
         self.assertIsNone(self._migrate_property(CaseSearchProperty(name='name', label={'en': 'Name'})))

